README.md

```diff
@@ -15,6 +15,7 @@ Official documentation on how to use the Azure App Configuration provider is ava
 
 * [.NET Core Console App](./examples/ConsoleApplication)
 * [ASP.NET Core Web App](./examples/ConfigStoreDemo)
+* [Azure Functions App in isolated process](./examples/FunctionAppIsolatedMode)
```

Comment:
I'm debating whether we should duplicate the examples here or we should direct users to https://github.com/Azure/AppConfiguration/blob/main/examples/README.md (and clean up all examples here). Thoughts?

Reply:
I think the examples here are more useful for developers of this library than consumers. I find it easier to build and test using these sample apps.

Reply:
Sure, if you find it useful.